### PR TITLE
PI-1012 added API test as end to end test

### PR DESCRIPTION
**Pull request by stevomcallister**

### CODE REVIEW

File: steps/api/epf/epf-api.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    return request.newContext({
+        baseURL: process.env.EPF_API,
+        extraHTTPHeaders: {
+            Accept: 'application/json',
+            Authorization: `Bearer ${token}`,
+        },
+    })
+}
+
+export async function epfContext(crn: string, eventNumber: string): Promise<any> {
```
Comment (marcus-bcl) [inline steps/api/epf/epf-api.ts]: I don't think it matters too much, but we're getting a warning here for using the `Any` type.  Is it worth creating an interface for the API response?

File: steps/api/epf/epf-api.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import { type APIRequestContext, expect, request } from '@playwright/test'
+import { getToken } from '../auth/get-token.js'
+
+async function getContext(): Promise<APIRequestContext> {
+    const token = await getToken()
```
Comment (marcus-bcl) [inline steps/api/epf/epf-api.ts:5]: I think this uses my personal dev client, I'll ask the auth team to add the EPF_CONTEXT role now

**Review by marcus-bcl — DISMISSED** [review]

File: steps/api/epf/epf-api.ts (hunk 2 above)
Comment (stevomcallister) [reply steps/api/epf/epf-api.ts:5]: I asked Matt to add them everywhere we need them so he might have done it

File: steps/api/epf/epf-api.ts (hunk 1 above)
Comment (stevomcallister) [reply steps/api/epf/epf-api.ts]: ok done

**Review by marcus-bcl — APPROVED** [review]

**Pull request merged**